scripts/build_lib.sh

```diff
@@ -19,6 +19,22 @@ toggle_failpoints() {
   mode="$1"
   if command -v gofail >/dev/null 2>&1; then
     run gofail "$mode" server/etcdserver/ server/storage/backend/
+    (
+      cd ./server
+      run go get go.etcd.io/gofail/runtime
+    ) || exit 2
+    (
+      cd ./etcdutl
+      run go get go.etcd.io/gofail/runtime
+    ) || exit 2
+    (
+      cd ./etcdctl
+      run go get go.etcd.io/gofail/runtime
+    ) || exit 2
+    (
+      cd ./tests
+      run go get go.etcd.io/gofail/runtime
```

Comment:
Why do we need to get `gofail/runtime` under `tests` ?

Reply:
Because tests import server.

Reply:
ack